examples/logistic/prediction_joint.py

```diff
@@ -10,17 +10,14 @@
 prediction_client_1 = PredictionClient('prediction-client-1', num_features // 2)
 result_receiver = prediction_client_0
 
-x_0 = tfe.define_private_input(prediction_client_0.player_name,
-                               prediction_client_0.provide_input)
-x_1 = tfe.define_private_input(prediction_client_1.player_name,
-                               prediction_client_1.provide_input)
+x_0 = prediction_client_0.provide_input()
+x_1 = prediction_client_0.provide_input()
```

Comment:
should be `prediction_client_1`

Reply:
since there's a bit more magic now, maybe it's worth adding some assertions to this example, just to show the user that eg `x_0` and `x_1` are private tensors. once we refactor we can also assert something like `x_0.owner == prediction_client_0.player`

Reply:
Up to you!